src/main/io/gps.c

```diff
@@ -491,11 +548,39 @@ static void ubloxSendPollMessage(uint8_t msg_id)
     ubloxSendMessage((const uint8_t *) &tx_buffer, 6);
 }
 
-static void ubloxSendNAV5Message(uint8_t model)
-{
+static void ubloxSendNAV5Message(uint8_t model) {
     ubxMessage_t tx_buffer;
+    /*if (gpsData.unitVersion == UBX_VERSION_M10) {
+        tx_buffer.payload.cfg_nav5m10.mask = 0xFFFF;
+        tx_buffer.payload.cfg_nav5m10.dynModel = model == 0 ? model : model + 1; //no model with value 1
+        tx_buffer.payload.cfg_nav5m10.fixMode = 3;
+        tx_buffer.payload.cfg_nav5m10.fixedAlt = 0;
+        tx_buffer.payload.cfg_nav5m10.fixedAltVar = 10000;
+        tx_buffer.payload.cfg_nav5m10.minElev = 5;
+        tx_buffer.payload.cfg_nav5m10.drLimit = 0;
+        tx_buffer.payload.cfg_nav5m10.pDOP = 250;
+        tx_buffer.payload.cfg_nav5m10.tDOP = 250;
+        tx_buffer.payload.cfg_nav5m10.pAcc = 100;
+        tx_buffer.payload.cfg_nav5m10.tAcc = 300;
+        tx_buffer.payload.cfg_nav5m10.staticHoldThresh = 0;
+        tx_buffer.payload.cfg_nav5m10.dgnssTimeout = 60;
+        tx_buffer.payload.cfg_nav5m10.cnoThreshNumSVs = 0;
+        tx_buffer.payload.cfg_nav5m10.cnoThresh = 0;
+        tx_buffer.payload.cfg_nav5m10.reserved0[0] = 0;
+        tx_buffer.payload.cfg_nav5m10.reserved0[1] = 0;
+        tx_buffer.payload.cfg_nav5m10.staticHoldMaxDist = 200;
+        tx_buffer.payload.cfg_nav5m10.utcStandard = ubloxUTCStandardConfig_int[gpsConfig()->gps_ublox_utc_standard];
```

Comment:
I think there is a critical bug here. For example if `gpsConfig()->gps_ublox_utc_standard == UBLOX_UTC_STANDARD_NTSC`

```c
ubloxUTCStandardConfig_int[gpsConfig()->gps_ublox_utc_standard];
ubloxUTCStandardConfig_int[UBLOX_UTC_STANDARD_NTSC];
ubloxUTCStandardConfig_int[7];  // array index out of bounds!!!
```
`ubloxUTCStandardConfig_int[]` has only 5 elements, so the FC crashes in this example, I reckon.

Reply:
It uses the integer value